sb.web.app/src/locales/en.json

```diff
@@ -153,7 +153,7 @@
         "purchase_request": "You have received a purchase request from",
         "go_to": ". Go to",
         "approve_or_decline": "to approve or decline.",
-        "have_sent": "have sent",
+        "have_sent": "has sent",
```

Comment:
Take a look at some of the more complex examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that can avoid these weird 'sentence fragments' lying around

Reply:
Okay.
I figured we were doing a full switch to English.
I'll chuck them into the en.josn and use that for all these scenarios

Reply:
We'll kind of have to hard-code some of these values in the code if we end up using this named formatting method.